From 555992ec9725494b487b2893f5d8e4f879bd4c27 Mon Sep 17 00:00:00 2001
From: Greg Campbell <gtcampbell@gmail.com>
Date: Thu, 5 Aug 2010 12:09:57 -0700
Subject: [PATCH] Add missing ActiveModel::Validations require

[#5311 state: resolved]

ActiveModel::Validations uses Hash#except, but does not require it from
ActiveSupport.  (This wasn't showing up in the tests, because it was
required in the helper, and was also required in
ActiveModel::Serialization).
---
 activemodel/lib/active_model/validations.rb |    1 +
 activemodel/test/cases/helper.rb            |    1 -
 2 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/activemodel/lib/active_model/validations.rb b/activemodel/lib/active_model/validations.rb
index 3407c59..37429f2 100644
--- a/activemodel/lib/active_model/validations.rb
+++ b/activemodel/lib/active_model/validations.rb
@@ -2,6 +2,7 @@ require 'active_support/core_ext/array/extract_options'
 require 'active_support/core_ext/array/wrap'
 require 'active_support/core_ext/class/attribute'
 require 'active_support/core_ext/hash/keys'
+require 'active_support/core_ext/hash/except'
 require 'active_model/errors'
 require 'active_model/validations/callbacks'
 
diff --git a/activemodel/test/cases/helper.rb b/activemodel/test/cases/helper.rb
index a32f114..a81584b 100644
--- a/activemodel/test/cases/helper.rb
+++ b/activemodel/test/cases/helper.rb
@@ -6,7 +6,6 @@ $:.unshift(lib) unless $:.include?('lib') || $:.include?(lib)
 require 'config'
 require 'active_model'
 require 'active_support/core_ext/string/access'
-require 'active_support/core_ext/hash/except'
 
 # Show backtraces for deprecated behavior for quicker cleanup.
 ActiveSupport::Deprecation.debug = true
-- 
1.7.0.4

